### ISSUE TITLE
TTR Not found

### ISSUE THREAD

**Issue by Megara1202** [kpreisser/MouseClickSimulator#37]
Says TTR not found and to make sure its running, tried running in admin and still nothing
Unless its just not compatible anymore?

**Comment by Cmarlowe132** [comment]
Would imagine that toontown rewritten has changed something in the recent update with how the game runs. The error message says "TTREngine.exe not found" when trying to run.

**Comment by atemerino** [comment]
> Would imagine that toontown rewritten has changed something in the recent update with how the game runs. The error message says "TTREngine.exe not found" when trying to run.

As far as I know you're using the 64-bit launcher which uses "TTREngine64.exe". Renaming every "TTREngine.exe" string would solve this.

**Comment by Cmarlowe132** [comment]
> 

Doing this did solve the issue of the simulator not finding the game, now I am running into the issue that the script is struggling to find where the bubbles are, and is either not seeing them at all or keeps seeing them in the same spot. Not really sure what to do about this one.

**Comment by Megara1202** [comment]
Wheres the string of text to edit the actual exe? cant seem to find it

**Comment by kpreisser** [comment]
Hi, thanks for the report! I changed the executable name for TT Rewritten to `TTREngine64`, so the error should no longer occur.

About the simluator not finding the fish bubbles, this could be caused by the current winter theme, which causes the bubbles to have a bit different colors. The colors are specified in the `bubbleColorRgb` attribute in the project.
I added a new project `Automatic Fishing - Punchline Place - Winter Theme.xml`, where I adjusted the color values to `20, 130, 118`, which seems to work for me.

Thanks!
